blakserv/ccode.c

```diff
@@ -2344,3 +2346,61 @@ blak_int C_MinigameStringToNumber(int object_id,local_var_type *local_vars,
 	
 	return ret_val.int_val;
 }
+
+bool IsAllowedConfigID(int config_id) {

```

Comment:
If you make server_config_whitelist a std::vector instead, this function could be one line (using the find() function).